src/test/java/uk/nhs/hee/tis/trainee/sync/service/PersonCachingTest.java

```diff
@@ -0,0 +1,198 @@
+/*
```

Comment:
It might be worth moving this package into the same place as the other caching tests, i.e. 
\tis-trainee-sync\src\test\java\uk\nhs\hee\tis\trainee\sync\cache

Reply:
I was unable to create a new TCSSyncService when I had this with the other caching test without making the TCSSyncService public, I thought it would be better not to make it public and keep the tests in the same area unless its ok to go public